content/applications/websites/website/web_design/building_blocks/inner_content_blocks.rst

```diff
@@ -0,0 +1,39 @@
+=============================
+Inner content building blocks
+=============================
+
+You can use the :guilabel:`Inner content` building blocks, such as the :guilabel:`Button`,
+:guilabel:`Image`, :guilabel:`Video`, or  :ref:`Social Media <inner_content_blocks/social_media>`
+blocks, to enhance your website's view by providing a structure and design for the content available
+on your web pages. Select a block from the :guilabel:`Inner content` section in the
+:guilabel:`Blocks` tab of the website builder, and drag and drop it onto your web page.
+
+.. image:: inner_content_blocks/inner_content_blocks.png
+  :alt: The inner content building blocks available in the website builder.
+
+.. seealso::
+  :doc:`Building blocks <../building_blocks>`
+
+.. _inner_content_blocks/social_media:
+
+Social Media block
+------------------
+
+Displaying your company's social media on your website is a good manner to grow brand awareness and
+improve :doc:`SEO <../../../website/pages/seo>`.
+
+Select the :guilabel:`Social Media` building block from the :guilabel:`Inner content` section and
+drag and drop it anywhere on your website, including in existing building blocks.
+
+.. image:: inner_content_blocks/social_media_block.png
+  :alt: The social media block.
+
+Once added to your web page, select the :guilabel:`Social Media` block, go to the
+:guilabel:`Customize` tab, and go to the :guilabel:`Social Media` section to edit it.
```

Comment:
Same as the general comment: this part could go into the config h2 and this paragraph could be part of the social media example

Reply:
As explained before, the idea is not to have a config section to be consistent with the other pages.